tools/cldr-apps/src/main/liberty/config/server.env.sample

```diff
@@ -0,0 +1,4 @@
+# copy this to server.env and set the password properly
```

Comment:
@btangmu if you copy this file to `server.env` (with appropriate DB changes) then you can start the server with `mvn -pl cldr-apps liberty:dev`  and it will fire up a dev server on port localhost:9080 until you hit control-C ..